applications/nrf_desktop/README.rst

```diff
@@ -798,6 +798,27 @@ By default, this period is set to 120 seconds.
     If a nRF Desktop device supports remote wakeup, the USB connected device goes to suspended state when USB is suspended.
     The device can then trigger remote wakeup of the connected host on user input.
 
+Configuration
+*************
+
+|config|
+
+Adding nRF21540 EK shield support
+=================================
+
+The nRF Desktop application can be used with the :ref:`ug_radio_fem_nrf21540_ek` shield, an RF front-end module (FEM) for the 2.4 GHz range extension.
+Any nRF Desktop HID application can be built with the shield support by passing the ``SHIELD`` parameter to the build command.
```

Comment:
Should we refer only to development kits here? The custom HW (e.g. mouse, kbd, dongle) cannot be used with shield.

Reply:
`nRF Desktop HID application configured for a development kit can be built ...`

Reply:
The shield cannot be used out of the box together with nRF desktop's custom hardware (like gaming mouse, keyboard and dongle)

Reply:
How about:
```
This, out of the box, method of enabling SHIELD support is only available for boards that are fitted with Arduino compatible connector.
```

Reply:
I suggest combining both:
```suggestion
The shield can be used with any nRF Desktop HID application configured for a development kit that is fitted with Arduino-compatible connector (see the :guilabel:`DK` tab in `Requirements`_).
This means that the shield support is not available for nRF Desktop's dedicated , such as ``nrf52840gmouse_nrf52840``, ``nrf52kbd_nrf52832``, or ``nrf52840dongle_nrf52840``.
To build the application with the shield support, pass the ``SHIELD`` parameter to the build command.
```